_projects/2023/100505771/100505771.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "Mexico's Export Dynamics from 1996 to 2020 by Sector"
+description: |
+  Replication and alternative version of the over time graph sector exports of Mexico by the Growth Lab at Harvard University.
+author: Valeria Contreras
+date: 2023-12-12
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+##INTRODUCTION
```

Comment:
A space is needed between the `##` symbols and the title. Also, please use sentence case for all titles, not upper-case.